orga/app/components/auth/login-form.hbs

```diff
@@ -20,43 +20,34 @@
 
   <form {{on "submit" this.authenticate}}>
     <p class="login-form__information">{{t "pages.login-form.mandatory-all-fields"}}</p>
+
     <div class="input-container">
-      <label for="login-email" class="label">{{t "pages.login-form.email"}}</label>
-      <Input
-        id="login-email"
+      <PixInput
+        @id="login-email"
+        @label={{t "pages.login-form.email"}}
+        name="login"
         @type="email"
         @value={{this.email}}
-        name="login"
-        class="input"
-        autocomplete="username"
+        {{on "focusout" this.validateEmail}}
```

Comment:
**suggestion**
Regarder si on peut enlever le `@value={{this.email}}` au profit de `on 'change'`